test/lexical/collections/_LexicalFullyFeatured/e2e.spec.ts

```diff
@@ -131,4 +136,39 @@ describe('Lexical Fully Featured', () => {
     const someButton = dropdownItems!.locator(`[data-item-key="bg-red"]`)
     await expect(someButton).toHaveAttribute('aria-disabled', 'false')
   })
+
+  async function uploadsTest(page: Page, mode: PasteMode, expectedFileName?: string) {
+    const filePath = path.resolve(dirname, './collections/Upload/payload.jpg')
```

Comment:
I would move this function to utils.ts. If the problem is that it needs access to the Payload instance, let's have LexicalHelpers receive it in the constructor.

Reply:
Well, I don't know if it would be in `utils` or where, but I would have moved it somewhere else, since this file can grow indefinitely and I'd like it to be as declarative and simple as possible.

But it's a minor detail. You make the final decision 😊